src/providers/mistral-ai/chatComplete.ts

```diff
@@ -53,6 +53,16 @@ export const MistralAIChatCompleteConfig: ProviderConfig = {
     param: 'safe_prompt',
     default: false,
   },
+  prompt: {
+    param: 'prompt',
+    required: true,
```

Comment:
`required` should be false as it is not required for normal mistral requests.

Reply:
done